en/simulation/README.md

```diff
@@ -159,32 +159,64 @@ The simulated camera is a gazebo plugin that implements the [MAVLink Camera Prot
 
 The same approach can be used by other simulators to implement camera support.
 
-## Running Simulation on the Remote Server 
+## Running Simulation on a Remote Server
 
-As had been mentioned at the beginning the simulation environment can be run on multiple computers on the same network. Unfortunately, it is a slightly complicated, because of the out of box configuration does not broadcast the PX4 UDP packets to external interfaces and packets are routed internally by default. 
-A solution is to enable brodcasting by [MAV_BROADCAST](../advanced/parameter_reference.md#MAV_BROADCAST) parameter to allow broadcast UDP packets to the local network or use a tunnel to connect computers together. 
+It is possible to run the simulator on one computer, and access it from another computer on the same network (or on another network with appropriate routing). 
+This might be useful, for example, if you want to test a drone application running on real companion computer hardware running against a simulated vehicle.
 
-Using the tunnel is a more flexible option because the computers are not required to sit on the same network and remote powerful simulation server can be used for example. 
+This does not work "out of the box" because PX4 does not route packets to external interfaces by default (in order to avoid spamming the network and different simulations interfering with each other). Instead it routes traffic internally - to "localhost".
 
-One, probably the easiest way to create the tunnel is the use of SSH tunneling options.  The tunnel itself could be created easily by running the following command on localhost.
+There are a number of ways to make the UDP packets available on external interfaces, as outlined below.
+ 
+
+### Enable MAV_BROADCAST
+
+Enable [MAV_BROADCAST](../advanced/parameter_reference.md#MAV_BROADCAST) to broadcast heartbeats on the local network.
+
+A remote computer can then connect to it by listening to the Simulation computer's IP address on the appropriate port (i.e. 14550 for *QGroundControl*).
```

Comment:
@bkueng Is this correct? Ie Say sim is running on IP a.b.c.d then before setting MAV_BROADCAST you could only access port 14550 internally, but afterwards you could access the sim from another computer by listenting on http://a.b.c.d:14550.

If not, how does it work?

Reply:
@bkueng @kaklik Thanks for help. OK, this is done modulo post above ^^^. If it is possible for a remote computer that knows the IP of the server computer to just connect to http://a.b.c.d:14550 then I should add that too.

Otherwise OK to publish?

Reply:
> Ah, so in broadcast every computer on the network gets messages on its port IP?

Yes

> If you did know the remote computer IP, could you look at http://a.b.c.d:14550. without the broadcast being enabled (this was how I thought you would connect to a server - localhost is exposed externally).

No, localhost is not exposed. By default the messages do not leave the localhost machine.

Yes, good to merge.

Reply:
Thank you very much @kaklik for the content, and @bkueng for the review. I think this is a pretty helpful addition.